ami/ml/post_processing/class_masking.py

```
@@ -0,0 +1,271 @@
+            score=max(scores),
+            scores=scores,
+            logits=logits,
+            detection=classification.detection,
+            timestamp=classification.timestamp,
+            terminal=True,
+            category_map=None,  # @TODO need a new category map with the filtered taxa
+            created_at=timestamp,
+            updated_at=timestamp,
+        )
+        if new_classification.taxon is None:
+            raise (ValueError("Classification isn't registered yet. Aborting"))  # @TODO remove or fail gracefully
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)